pyscf/md/__init__.py

```diff
@@ -45,4 +45,4 @@ def set_seed(seed):
 from pyscf.md import integrators, distributions
 
 NVE = integrators.VelocityVerlet
-
+NVT = integrators.NVTBerendson
```

Comment:
I don't think we should make the Berendson thermostat the default for NVT given the discussion

Reply:
Changed to NVT_Berendson!